app/views/users/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<% if @user.errors.any? %> <div id="error_explanation"> <h2><%= pluralize(@user.errors.count, "error") %> prohibited this post from being saved:</h2> <ul> <% @user.errors.full_messages.each do |msg| %> <li><%= msg %></li> <% end %> </ul> </div> <% end %> 
+<%= form_for @user do |f|%>
+
+<label for="username">Username</label>
+<%= f.text_field :username%>
+<br>
+<label for="email">Email</label>
+<%= f.email_field :email%>
+<br>
+<label for="password">Password</label>
+<%= f.password_field :password%>
+<br>
+<%= f.submit "Submit"%>
+<%end%>
+
+<%# the text below are the examples of the old versions of forms we did before the updated one above, we kept it as advised by TheOdinProject%>
+
+
+<% if false%>
+<%= form_tag(users_path, method: "POST") do %>
+     <%= label_tag(:username, "Username:") %>
+     <%= text_field_tag(:username) %><br><br>
+     <%= label_tag(:email, "email:") %>
+     <%= email_field_tag(:email) %><br><br>
+     <%= label_tag(:password, "password:") %>
+     <%= password_field_tag(:password) %><br><br>
+     <%= submit_tag("Submit")%>
+<% end %>

```

Comment:
This form should be commented out.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

Use the link as a reference.